packages/core/admin/admin/src/pages/Settings/pages/Users/tests/EditPage.test.tsx

```diff
@@ -8,7 +8,7 @@ describe('Users | EditPage', () => {
       initialEntries: ['/settings/users/1'],
     });
 
-    expect(screen.getByRole('heading')).toHaveTextContent('Edit user');
+    await waitFor(() => expect(screen.queryByText('Loading content.')).not.toBeInTheDocument());
```

Comment:
findByText?

Reply:
I think `findByText` would fail because we're asserting that it doesn't exist in the dom?